.config/karma.conf.js

```diff
@@ -0,0 +1,84 @@
+module.exports = function(config) {
+    config.set({
+
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: './../',
+
+
+        // frameworks to use
+        // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+        frameworks: ["qunit"],
+
+
+        // list of files / patterns to load in the browser
+        files: [
+            "node_modules/jquery/dist/jquery.js",
+            "tests/lib/helpers.js",
+            "dist/all.css",
+            "tests/fixtures/**/*.html",
+            "tests/data/metrics.js",
+            "tests/**/*-qunit.js"
```

Comment:
I'm assuming that in the different themes, the tests and fixtures will be reused with a different metrics object, is that correct? If so, what will be the configuration for that?